bigquery/google/cloud/bigquery/magics.py

```diff
@@ -320,6 +320,22 @@ def _run_query(client, query, job_config=None):
     default=None,
     help=("Project to use for executing this query. Defaults to the context project."),
 )
+@magic_arguments.argument(
+    "--max_results",
+    default=None,
+    help=(
+        "Maximum number of rows in dataframe returned from executing the query."
+        "Defaults to returning all rows."
+    ),
+)
+@magic_arguments.argument(
```

Comment:
Looks like these arguments got doubled up. (Maybe the two commits thing, we noticed when rebasing?)

Reply:
That's probably what happened (moral of the story, always run tests before pushing). Removed the duplicate in 147e43d